src/math/Frustum.js

```diff
@@ -161,7 +163,10 @@ class Frustum {
 	intersectsSprite( sprite ) {
 
 		_sphere.center.set( 0, 0, 0 );
-		_sphere.radius = 0.7071067811865476;
+
+		const offset = _center.distanceTo( sprite.center );
```

Comment:
`sprite.center` is a `Vector2`.

And what are the units of `offset` in your view?

And what about the perspective camera use case when `sizeAttenuation` is `false`?

Reply:
Sorry, @Mugen87, I had a confusing typo in my prior post. Fixed it -- but after you posted.

Reply:
I always viewed `sprite.center` as a UV, but in spite of that, I think your calculation is correct for the use case you are considering.

Unfortunately, the code is easy to misread because there are three "centers".